cmd/init.go

```diff
@@ -103,10 +105,15 @@ dapr init --from-dir <path-to-directory>
 				dockerNetwork = viper.GetString("network")
 				imageRegistryURL = viper.GetString("image-registry")
 			}
-			if fromDir != "" {
+			// If both --image-registry and --from-dir flags are given, error out saying only one can be given.
+			if len(strings.TrimSpace(imageRegistryURL)) != 0 && len(strings.TrimSpace(fromDir)) != 0 {
+				print.FailureStatusEvent(os.Stderr, "both --image-registry and --from-dir flags cannot be given at the same time")
+				os.Exit(1)
+			}
+			if len(strings.TrimSpace(fromDir)) != 0 {
 				print.WarningStatusEvent(os.Stdout, "Local bundle installation using from-dir flag is currently a preview feature.")
```

Comment:
Should we add that it's only available from XYZ version like `--image-registry`?